src/ai_provider/ollama_ai_provider.py

```
@@ -0,0 +1,132 @@
+"""
+This module provides an AI provider for interacting with the Ollama API.
+"""
+
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)